hack/fetch-ext-bins.sh

```diff
@@ -26,7 +26,7 @@ if [ -n "$TRACE" ]; then
   set -x
 fi
 
-k8s_version=1.22.1
+k8s_version=1.23.5
```

Comment:
should it be 1.24.something?

Reply:
ok, no kubebuilder for 1.24 yet, ignore this

Reply:
We always lag behind several versions with those envtest kubebuilder tools (On some repos we where even on 1.19). As they release them significantly later unfortunately. That's why we were still on 1.22 on the previous release. For now we are stuck on 1.23.x.